stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            for map in map_list:
</pre><p> </p><p>Using <code>map</code> as a variable name is kinda nasty as it's a built-in function. Maybe <code>map_name</code>? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>I'd agree. As I mention earlier if the terminology can be switched throughout to 'game', then that might help here.</p>